tests/security/pam/pam_login.pm

```diff
@@ -17,15 +18,22 @@ sub run {
     # Define the user and password, which are already configured in previous milestone
     my $user = 'suse';
     my $passwd = 'susetesting';
+    my $pam_login = '';
+    my $pam_login_tw = '';
 
     # Modify the login/sshd files to set the PAM authentication
     my $deny_user_file = '/etc/deniedusers';
     my $pam_sshd = '/etc/pam.d/sshd';
-    my $pam_login = '/etc/pam.d/login';
     my $pam_sshd_bak = '/tmp/sshd_bak';
     my $pam_login_bak = '/tmp/login_bak';
     my $pam_sshd_tw = '/usr/etc/pam.d/sshd';
-    my $pam_login_tw = '/usr/etc/pam.d/login';
+    if (is_sle || is_leap) {
+        $pam_login = '/etc/pam.d/login';
+        $pam_login_tw = '/usr/lib/pam.d/login';
+    } else {
+        $pam_login = '/lib/pam.d/login';
```

Comment:
That has a very negative side effect...

this code is used further down in:

```
my $ret_login = script_run("[[ -e $pam_login ]]");
…
    if ($ret_login != 0) {
        script_run "cp $pam_login_tw $pam_login";
    }
…
```

So this now copies a file from `/usr/lib/pam.d` to `/lib/pam.d`, which is, on TW, even the same. 
Even worse, that file is further down the line being modified.

This is definitively not what an admin is supposed to do; the admin is supposed to instanciate his template in /etc/pam.d and modify it there.

Reply:
https://openqa.opensuse.org/tests/2536618#step/pam_login/23 here we can see it modifying a file in /lib

Reply:
Thanks for the comment, yes it makes sense, I will fix it!